src/Smesh.jl

```diff
@@ -1,33 +1,114 @@
 module Smesh
 
-using Preferences: @load_preference, @has_preference 
+using Preferences: @load_preference
 using smesh_jll: smesh_jll
 
-export build_delaunay_triangulation
+export build_delaunay_triangulation, delaunay_compute_neighbors, build_polygon_mesh, voronoi_compute_neighbors
 
 const libsmesh = @load_preference("libsmesh", smesh_jll.libsmesh)
 
-
 """
 """
 function build_delaunay_triangulation(data_points; shuffle = false, verbose = false)
```

Comment:
You should check if this function is type stable. It should be, but if not, you should take a look

Reply:
Thanks. Yes, it is.